core/src/main/java/com/github/mizool/core/converter/DtoConverters.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2018 incub8 Software Labs GmbH
+ * Copyright 2018 protel Hotelsoftware GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.mizool.core.converter;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+
+import lombok.experimental.UtilityClass;
+
+import com.github.mizool.core.GuavaCollectors;
+
+@UtilityClass
+public class DtoConverters
```

Comment:
We discussed renaming this to `CollectionConverters`, but I'm no longer sure whether that name is a good fit. Our naming scheme seems to imply such a class offers utility methods for classes that are called "collection converter", but we have no such thing. In fact, this class itself comes close: we might turn it into an instantiable class and call it `CollectionConverter`. Then again, it would be an unusual converter as its methods require passing the "element converter". Also, instantiating such a CollectionConverter _and_ the converter for the elements would probably _add_ boilerplate at the call sites, and the goal was to _reduce_ it.

So I tend towards leaving it a utility class, but the name is weird.
@voons @JanDoerrenhaus @schroenser: Any opinions?

Note: the most elegant solution would probably be to introduce a `Converter` interface which adds default methods for `List`/`Set`/`Map` to each converter that implements the interface. But I think this is way out of scope now.

Reply:
@bannmann Maybe just `Converters`?

Reply:
I can't think straight, but `Converters` sounds good, for the time being.